bible.php

```diff
@@ -126,6 +126,11 @@ static function translationPrepare($translation, &$text)
                 $text = preg_replace('/<sup>([0-9]{1,2})<\/sup>/', '<p>$1', $text);
                 $text = preg_replace('/<sup>([0-9]{1,2})[-\x{2013}]([0-9]{1,2})<\/sup>/u', '<p>$1-$2', $text); // unicode minus
                 break;
+            case "NET":
+                $text = str_replace("<br>\n", ' ', $text);
+                $text = preg_replace('/<bqverse ([0-9]{1,2})>([0-9]{1,2})/', '<p>$1', $text);
```

Comment:
What if there are more than 100 verses? Shouldn't it be like this?:
```suggestion
                $text = preg_replace('/<bqverse ([0-9]{1,3})>([0-9]{1,3})/', '<p>$1', $text);
```